packages/itmat-interface/src/graphql/resolvers/userResolvers.ts

```diff
@@ -228,7 +228,7 @@ export const userResolvers = {
             });
         },
         createUser: async (__unused__parent: Record<string, unknown>, args: any): Promise<IGenericResponse> => {
-            const { username, type, realName, email, emailNotificationsActivated, password, description, organisation }: {
+            const { username, type = Models.UserModels.userTypes.STANDARD, realName, email, emailNotificationsActivated, password, description, organisation }: {
```

Comment:
i think this is not necessary because of graphql schema type check

Reply:
if type is empty it will never reach this function. it will be rejected by graphql server